objectRecognitionSource/README.md

```diff
@@ -504,6 +504,24 @@ These files are now fetched from an Amazon S3-based maven repository, and will b
 
 Previously, `yolo.pb` file was downloaded as part of the build, but the `coco.names` file was stored in Git.  That file is still in git (but likely to be removed soon).  We now obtain all the model files from the same place -- so none of this information will be in Git.  Keeping the information together is a safer way to go about things.
 
+### <a name="noNet" id="noNet"></a>No Processor
+
+The purpose of this implementation of the Neural Net Interface is to save images without doing any processing. This allows a 
+user to capture frames from their video or image source, and save them with VANTIQ, locally, or both. There is absolutely **no 
+processing** done to the captured frames, they are saved as is.
+
+The Extension Source can be setup for polling, for queries, or for both.
+
+*   outputDir: Optional. Config and Query. The directory in which the images (object boxes included) will be placed.
+    Images will be saved as "&lt;year&gt;-&lt;month&gt;-&lt;day&gt;--&lt;hour&gt;-&lt;minute&gt;-&lt;second&gt;.jpg"
+    where each value will zero-filled if necessary, e.g. "2018-08-14--06-30-22.jpg". If pollTime is set under 1000 (1 
+    second), images with the same name will be followed by a count in parentheses, e.g. "2018-08-14--06-30-22(1).jpg", "2018-
+    08-14--06-30-22(2).jpg".
+*   fileName: Optional. Query only. The name of the file that will be saved. Defaults to
+    "&lt;year&gt;-&lt;month&gt;-&lt;day&gt;--&lt;hour&gt;-&lt;minute&gt;-&lt;second&gt;.jpg" if not set. As stated above, 
+    redundant filenames will be followed by a count in parentheses.
```

Comment:
Presumably, if I set the filename to "fred", I will simply get a million files named fred, fred(1), ..., fred(1000000) or something like that?